packages/@aws-cdk/core/README.md

```diff
@@ -15,8 +15,236 @@
 ---
 <!--END STABILITY BANNER-->
 
-This library includes the basic building blocks of
-the [AWS Cloud Development Kit](https://github.com/awslabs/aws-cdk) (AWS CDK).
+This library includes the basic building blocks of the [AWS Cloud Development Kit](https://github.com/awslabs/aws-cdk)
+(AWS CDK).
+
```

Comment:
Can you take a pass at thinking through this README? This is the core library and this readme is actually the most important one? I feel it could use some TLC. A bit of discussion and/or links to user manual around constructs, stacks and apps, etc.

Reply:
As discussed, this will be done in a second pass, so as not to delay the nice new content.